src/helpers/isPointerEventEnabled.ts

```diff
@@ -0,0 +1,26 @@
+import { ReactTestInstance } from 'react-test-renderer';
+import { getHostParent } from './component-tree';
+
+// pointerEvents controls whether the View can be the target of touch events.
```

Comment:
Pls replace with TSDocs comment, so that VSCode will show intellisense on that method:
```/** ... */```